WordPress/gutenberg
# [RNMobile] Update image size UI control

## Description

Updates the mobile SelectControl (currently only used on mobile in the Image Block) to use a Cycle Button style UI (CyclePickerCell) instead of nested BottomSheets (PickerCell).

## How has this been tested?
See Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1574
In order to test with image urls updating correctly via api requests, this was tested in WordPress iOS and WordPress Android 
1. Open an article in the editor with an image.
2. Click the gear button to open image settings
3. Click the menu row labeled "Size" multiple times. 
4. Note the menu size value and Image within the block are updated simultaneously. 

## Screenshots
![size-changes](https://user-images.githubusercontent.com/1103838/71146730-5e891380-21f4-11ea-9eaf-6d66be5a7a34.gif)

## Types of changes





- Introduces cycle button UI for updating the size of an image from the BottomSheet menu in an Image Block. 
- Removes duplicated default image size declarations and instead utilizes image sizes from the editor settings data store. 


## Checklist:
